gapic/src/main/com/google/gapid/views/SettingsDialog.java

```diff
@@ -156,6 +163,14 @@ protected void configureDialog(FileDialog dialog) {
       };
       adbPath.addBoxListener(SWT.Modify, adbListener);
       adbListener.handleEvent(null);
+
+      SelectionListener allowUpdateChecksListener = new SelectionAdapter() {
```

Comment:
```
allowUpdateChecks.addListener(SWT.Selection,
    e -> allowUpdateCheckPrereleases.setEnabled(allowUpdateChecks.getSelection()));
```

Also consider clearing the checkbox if the auto-update checkbox is cleared.

Reply:
done